File: server/Models/Product.cs
namespace Monogram.Models;

public class Product
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public int Price { get; set; }
    public string ImgUrl { get; set; }
    public string ImgUrl2 { get; set; }
    public string ImgUrl3 { get; set; }
}

File: client/src/models/Product.js

export class Product {
    constructor(data) {
        this.id = data.id
        this.createdAt = data.createdAt
        this.updatedAt = data.updatedAt
        this.name = data.name
        this.description = data.description
        this.price = data.price
        this.imgUrl = data.imgUrl
        this.imgUrl2 = data.imgUrl2
        this.imgUrl3 = data.imgUrl3
        // NOTE This is how the products know where to be put on the shop page.
        this.category = data.category
    }
}